docs/Commands.md

```diff
@@ -75,11 +81,20 @@ Indexing books that have a "categories" field in which semicolons separate the v
 FT.CREATE books-idx ON HASH PREFIX 1 book:details FILTER SCHEMA title TEXT categories TAG SEPARATOR ";"
 ```
 
+Indexing a JSON document using JSON Path:
+
+```sql
+FT.CREATE idx ON JSON SCHEMA $.title AS title TEXT $.categories AS categories TAG SORTABLE
```

Comment:
I suggest to make sure this is auto tested exactly as documented (with search/aggregate command)
(even if a similar test exists, e.g., test_json.py:testDemo)